migrations/V20__Add_Initial_Submitted_At_Biz_Case.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE business_case ADD COLUMN initial_submitted_at timestamp with time zone;
```

Comment:
Is there a reason we're using `timestamp with time zone` here? Are we actually utilizing the time zone features?

Reply:
It's just good practice to keep all the data.